Camille/gen/dotUtils.js

```diff
@@ -37,7 +37,7 @@ function normalizePropName(propName, schemaName, value) {
   return name;
 }
 
-function stringifyType(prop, endpoint = null, nullable = false) {
+function stringifyType(prop, name = '', endpoint = null, nullable = false) {

```

Comment:
Appears `name` param is no longer needed because of `x-enum`?

Reply:
Forgot about that one, will remove it